mParticle-Apple-SDK/Utils/MPHasher.cpp

```diff
@@ -11,7 +10,7 @@ namespace mParticle {
         string lowerCaseStringToHash = stringToHash;
         transform(lowerCaseStringToHash.begin(), lowerCaseStringToHash.end(), lowerCaseStringToHash.begin(), ::tolower);
         
-        int hash = 0;
+        unsigned int hash = 0;
```

Comment:
I know this was original just `int`, but since we're changing the type anyway, I'd like to see a `uintXX_t` type just so we're explicit about the size. Easy way to find would be to add a log statement printing the sizeof int to confirm it then use that bit size.

Also, if it doesn't hurt anything, maybe we can specify the return type as well, but if that's too invasive (also modifying various other places, then we can skip that and clean this up more later).

Reply:
Actually let's look into this kind of thing later when we do a bigger refactor.